### ISSUE TITLE
PulseEffects tray icon - bypass via bash or dconf

### ISSUE THREAD

**Issue by boomshop** [wwmm/pulseeffects#589]
Hey Wellington,

here's the tray icon we've been talking about:

https://github.com/boomshop/pulseffectstray

AFAIK currently there's no bypass option until users create an empty preset called `Bypass`. It would be cool to set/get a full bypass (in- and outputs) state, like getter:

```
pulseeffects -b
> [0|1]
```
and setter:
```
pulseeffects -b 1
> Bypass enabled.
```
or:
```
pulseeffects -b 0
> Bypass disabled.
```
Maybe you prefer dconf for this which wouldn't matter to me.

This would be a cool feature for left-clicking the icon (and the icons color indeed).

Best
Markus

**Comment by wwmm** [comment]
Ok. I think that in this case adding a new command line option will be better as people could benefit from this on other situations. I will do that this week after adding your tutorial to PE manual.

**Comment by boomshop** [comment]
Great, thanks in advance! I'll make some changes to the tutorial and send it to you tonight (which is your today).

**Comment by wwmm** [comment]
Implemented! I decided to also add a button in the interface(top left corner)
![Screenshot from 2019-11-26 15-09-15](https://user-images.githubusercontent.com/16225172/69660837-efd6f100-105f-11ea-933a-590866581157.png)

In the end it required more work than I thought it would. Just touching each plugin switch key it would not be enough as we would need to keep track of which were or not enabled when disabling the bypass. As all plugins are inside a container(a GStreamer bin) removing this container from the pipeline would be more efficient.

For some reason GTK did not like the idea of not passing a parameter to boolean command line types. So the parameter is an integer that can have 3 values. `1` for enable, `2` for disable and `3` to query the state. Take a look at `pulseeffects --help`
